lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -111,6 +111,35 @@ function findDifference(path, actual, expected) {
     const keyPath = path + keyAccessor;
     const expectedValue = expected[key];
 
+    if (key === '_includes') {
```

Comment:
This needs an README update. and should probably be a separate PR.

Reply:
It was a separate PR, are you saying you want to merge https://github.com/GoogleChrome/lighthouse/pull/13614 separately?